src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/FailedProxyCache.cs

```diff
@@ -36,7 +36,7 @@ internal sealed class FailedProxyCache
 
         // This lock can be folded into _nextFlushTicks for space optimization, but
         // this class should only have a single instance so would rather have clarity.
-        private SpinLock _flushLock = new SpinLock();
+        private SpinLock _flushLock = new SpinLock(enableThreadOwnerTracking: false); // mutable struct; do not make this readonly
```

Comment:
I really wish non-copyable value types was a thing (or at minimum, the compiler provided information about all copies so an analyzer could make it effectively non-copyable).